requirements.txt

```diff
@@ -18,6 +18,9 @@ gdown
 gensim==3.8.3
 sklearn
 gdown
+readability-lxml
+bs4
+orjson
```

Comment:
Any particular reason for using this one instead of just use python's built-in `json` package?